lnd.go

```diff
@@ -164,10 +165,16 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 			err)
 	}
 
-	mkErr := func(format string, args ...interface{}) error {
-		ltndLog.Errorf("Shutting down because error in main "+
-			"method: "+format, args...)
-		return fmt.Errorf(format, args...)
+	mkErr := func(msg string, err error, attrs ...slog.Attr) error {
+		ltndLog.ErrorS(ctx, "Shutting down due to error in main "+
+			"method", err, attrs)
+
+		fmtStr := msg + ": %w"
+		for range attrs {
+			fmtStr += " %s"
+		}
+
+		return fmt.Errorf(fmtStr, err, attrs)
```

Comment:
Need to use spreading operator (`...`) here, otherwise it will look like this: `error creating wallet config: unable to unlock macaroons: invalid password%!(EXTRA []slog.Attr=[])`

Reply:
ah indeed - fixed & tested  now 👍